src/main/java/capstone/mfslbackend/model/Game.java

```diff
@@ -0,0 +1,37 @@
+package capstone.mfslbackend.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.*;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Objects;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@ToString
+@Data
```

Comment:
@ Data is not great in terms of performance when used at the same time as @ Entity, so it should be replaced with @ Getter and @ Setter